.github/workflows/ci_maven.yml

```diff
@@ -0,0 +1,55 @@
+# This is a basic workflow to help you get started with Actions
```

Comment:
It's a bit more than that ;). Can change for something that references the tool?

Reply:
Updated